dkron/ui.go

```diff
@@ -40,14 +40,17 @@ func (h *HTTPTransport) UI(r *gin.RouterGroup) {
 			if err != nil {
 				log.Error(err)
 			}
-			totalJobs := len(jobs)
-			successfulJobs := 0
-			failedJobs := 0
+			var (
+				totalJobs = len(jobs)
+				successfulJobs, failedJobs, pristineJobs int
+			)
 			for _, j := range jobs {
 				if j.Status == "success" {
 					successfulJobs++
-				} else {
+				} else if j.Status == "failed" {
 					failedJobs++
+				} else {
+					pristineJobs++
```

Comment:
The term "pristine" is confusing to me. To me it means 'unscathed', 'not worn by use'. I had to look through the code to see what the intent is, to discover that it denominates untriggered jobs.
IMHO using 'untriggered' is a lot more concise.

Reply:
Personally, I have used to declare something untouched (without execution/status changes ) but to be more clear, I will change to "untriggered".

Reply:
Done